inc/common/cloudflare-flexible-ssl.php

```diff
@@ -31,8 +31,13 @@ function rocket_is_cloudflare() {
  * @return bool
  */
 function rocket_is_cf_ip() {
+	if ( ! isset( $_SERVER['REMOTE_ADDR'] ) ) {
+		return false;
+	}
+
 	// Store original remote address in $original_ip.
-	$original_ip = filter_input( INPUT_SERVER, 'REMOTE_ADDR', FILTER_VALIDATE_IP );
+	$original_ip = filter_var( wp_unslash( $_SERVER['REMOTE_ADDR'] ), FILTER_VALIDATE_IP );
+
 	if ( ! isset( $original_ip ) ) {
```

Comment:
`filter_var()` doesn't return `null`, but `false`.
```suggestion
	if ( empty( $original_ip ) ) {
```